content/api/integrations/pagerduty.md

```diff
@@ -27,14 +27,14 @@ Configure your Datadog-PagerDuty integration directly through the Datadog API.
     * **`service_key`** [*required*]:
         Your Service name associated service key in Pagerduty.
 
-* **`subdomain`** :
+* **`subdomain`** [*optional*, *default*=**None**]:
     Your PagerDuty account's personalized subdomain name.
 
-* **`schedules`** :
+* **`schedules`** [*required*]::
     Array of your schedule URLs, e.g:
      `["https://my-pd.pagerduty.com/schedules#PCPYT4M", "https://my-pd.pagerduty.com/schedules#PKTPB7P"]`
 
-* **`api_token`** :
+* **`api_token`** [*required*]:
```

Comment:
@l0k0ms I feel that tagging these arguments as `required` is a bit misleading. Its only required when creating a new configuration. This has caused customers to mistakenly update values in an existing configuration. My solution was to add a note `All arguments are required when creating (`PUT`) a new PagerDuty configuration` and removing `required`. What are your thoughts on this? How can we make this clearer?

Reply:
🤔 I missed that, it might be the best way to go indeed, let me update the PR